onadata/apps/logger/migrations/0064_auto_20210304_0314.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.16 on 2021-03-04 08:14
+
+from django.db import migrations
+
+from onadata.apps.logger.models import XFormVersion, XForm
+from onadata.apps.api.tools import create_xform_version
+
+
+def create_initial_xform_version(apps, schema_editor):
+    """
+    Creates an XFormVersion object for an XForm that has no
+    Version
+    """
+    for xform in XForm.objects.filter(
+            downloadable=True,
+            deleted_at__isnull=True):
```

Comment:
Should we use a queryset iterator here? For a large number of forms, these may have adverse effects on a production system.